packages/create/src/recipes/nuxt/template/dbschema/default.esdl

```diff
@@ -0,0 +1,38 @@
+using extension auth;
+
+module default {
```

Comment:
In general, I think we should have these recipes be starting point for any kind of app, not really an example app, if that makes sense. I'd expect that the schema here is empty, and that we add the auth extension here conditionally based on options set by the user.

To that end, if you want to design a small single page that just scaffolds some _very_ basic layout and styles, definitely do that. Probably something closer to https://github.com/nuxt/starter/tree/v3 .

Reply:
Sure, sounds great.

I started doing the "most minimal" Nuxt starter, but for Nuxt it just looks like adding `nuxt-edgedb-module` to their modules array.

The rest of it is just described in [`nuxt-edgedb-module`](https://github.com/Tahul/nuxt-edgedb-module).

Should I push that instead?

Reply:
Sorry for the delay in getting back to you here.

I'm not sure I understand the question but what I expect to see in these recipes is a totally empty `default.esdl` file, not even including the `using extension auth;` expression even. No migrations, no query builder code generator output, etc. These recipes should represent how you'd start day 0 on a new project, not as an example project, if that makes sense. The UI/app itself should also be minimal so that developers can quickly check that everything is working right, but nothing else.

Does that make sense?